sites/fast-creator/specs/site.spec.md

```diff
@@ -19,4 +20,88 @@ Libraries are available in the component tab in the left pane, they can be enabl
 
 Each library will be defined by a configuration which includes the web components themselves, data examples that will be used when a component is added, the tag name of the components and the display name. See the [typings](./app/configs/typings.ts) file for the interface.
 
-When a user adds a library to a project, this library configuration will be dynamically imported.
+When a user adds a library to a project, this library configuration will be dynamically imported. Available libraries will be referenced in a dictionary file, this file will be used to create a UI.
```

Comment:
```suggestion
When a user adds a library to a project, the library configuration will be dynamically imported. Available libraries will be referenced in a dictionary file, which will be used to create a UI.
```

Took a stab at what I think you mean that reads a bit more clearly to me. If it's not accurate please disregard

Reply:
It's accurate, I just pushed a change that updates it to something similar.